conversion/TemplateConverter.py

```diff
@@ -0,0 +1,260 @@
+        # adding jijnja filter to replace output with empty string
+        replace ="{{"+replace+"|replace("+replace+",'')}}"
+        self.fileLines[lineNr] = self.fileLines[lineNr].replace(changeablePart, replace)
+
+    def convertOneLineIf(self,lineNr):
+        cleanText = self.cleanHTMLTags(lineNr)
+        changeablePart = re.search("#if.+then.+else.+#|#if.+then.+else.+", cleanText).group(0)
+        changeableParts=changeablePart.split(" then ")
+        changeableParts[0]=changeableParts[0].replace("#if","{% if").replace("$","")+"%}"
+        changeableParts[1]=changeableParts[1].replace(" else ","{% else %}")
+        if changeablePart.endswith("#"):
+            changeableParts[1]=changeableParts[1][:-1]+"{% endif %}"
+        else:
+            changeableParts[1]=changeableParts[1]+"{% endif %}"
+        replacement=changeableParts[0]+ changeableParts[1]
+        self.fileLines[lineNr] = self.fileLines[lineNr].replace(changeablePart,  replacement)
+
+    # has to be called after all for statments are converted
+    # TODO: add support for other statments
+    def convertSlurp(self, lineNr):
+        self.fileLines[lineNr] = self.fileLines[lineNr].replace("#slurp",  "")
+        lineNrEnd=lineNr
+        lineNrStart=lineNr
+        while("{% endfor" not in  self.fileLines[lineNrEnd]):
+            lineNrEnd+=1
+            if(lineNrEnd>lineNr+10):
+                print("Failed to convert slurp")
+                break
+
+        while ("{% for " not in self.fileLines[lineNrStart]):
+            lineNrStart -= 1
+            if(lineNrEnd<lineNr-10):
+                print("Failed to convert slurp")
+                break
+
+        self.fileLines[lineNrEnd] = self.fileLines[lineNrEnd].replace("{% endfor",  "{%- endfor")
+        self.fileLines[lineNrStart] = self.fileLines[lineNrStart].replace("%}",  "-%}")
+
+    # has to be callled before converPlaceHolder
+    def convertDanimcalPlaceHolder(self, lineNr):
+        changeablePart = re.search("\${.*}",  self.fileLines[lineNr]).group(0)
+        replace = lambda x: x.replace("${", "{{") + "}"
+        self.fileLines[lineNr] = self.fileLines[lineNr].replace(changeablePart, replace(changeablePart))
+
+    # python code importation is not supported
+    # moving it from scrip and deleting it from template
+    def convertImport(self, lineNr):
+        importKey="importedFrom"
+        fromImports = re.search("#from (.+) import(.+)",  self.fileLines[lineNr])
+        if(fromImports):
+            self.pythonConversions.update({fromImports.group(2):{importKey:fromImports.group(1)}})
+        imports=re.search("#import (.+)",  self.fileLines[lineNr])
+        if (imports):
+            self.pythonConversions.update({imports.group(1):{importKey:None}})
+        # clean the line
+        self.fileLines[lineNr]=""
+
+
+
+if __name__=="__main__":
+    fileLines=FileReader().readFile("/home/adelina/PycharmProjects/testCheetah/quote/qoute.tmpl")
```

Comment:
No hardcoded paths please. It would be much better if you'll add a local area with template examples against which you'll test your code.